packages/keycodes/src/index.js

```diff
@@ -216,3 +216,15 @@ export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
 		return event.key === character;
 	};
 } );
+
+/**
+ * Determine if an event is a "delete" keyboard event.
+ * @param {SyntheticEvent} event A synthetic keyboard event.
+ * @return {boolean} True if the event is a keyboard event, otherwise false.
+ */
+export const isDeleteKeyboardEvent = ( { keyCode, ctrlKey } ) => (
+	keyCode === DELETE ||
+	keyCode === BACKSPACE ||
+	// On Apple devices, treat control + d as a delete event
+	( isAppleOS && ctrlKey && keyCode === 68 )
```

Comment:
I _think_ this would be ok to apply to all environments, but I don't have a non-Apple computer on which to test.

Reply:
It would be cool if we could look at the `input` event instead and detect `deleteContentForward` and `deleteContentBackward`. This method would be more robust.

Reply:
I'll try those out, thanks!

Reply:
Thanks! The benefit of using that is that the OS doesn't matter. You could even have custom shortcuts and it would still work.

Reply:
It looks like `InputEvent`s don't get fired at the boundaries :-/

I tested like this:
`document.querySelector( '.editor-rich-text__editable' ).addEventListener( 'input', e => console.log( e ) );`

Events fire when inserting & deleting actually happens, but not if the keypress events don't actually result in an input change.

Is there another way we can access those properties?

Reply:
Could we do something like `const D = 'D'.charCodeAt( 0 );`?

Reply:
Updated w/ this approach (& to add <kbd>ctrl</kbd>+<kbd>h</kbd> support)